CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
    - FCoE network
    - Firmware bundles
    - Firmware drivers
+   - Interconnect
```

Comment:
Like a few of the other PRS, we'll want to add this to a new `3.1.0` section above, since 3.0.0 has already been released.